android/build.gradle

```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlin_version = '1.3.71'
+    ext.kotlin_version = '1.6.10'
```

Comment:
I have to updated these values because of build crashed with this message:

`your project requires a newer version of the kotlin gradle plugin`